sources/__tests__/edge-cases.test.ts

```diff
@@ -28,17 +28,31 @@ describe("edge cases and special scenarios", () => {
       },
     });
 
-    await workspace.yarn.add("typescript");
```

Comment:
This line simply runs `yarn add typescript` without using the catalog protocol. If it had been `yarn add typescript@catalog:`, the existing test would have failed to match this regex due to the double-patching issue.